### PR TITLE
Add cterm support

### PR DESCRIPTION
**Pull request by gko**
Second attempt. `Vim` on the left and `neovim` on the right. There are for sure some colors that might be slightly off. And you have to apply the color profile within `iTerm`. Alternatively we can define black color within palette as our background color. That is what @chriskempson did:
https://github.com/chriskempson/base16-shell/blob/master/scripts/base16-material-palenight.sh#L6

I also didn't manage to make work the italics in vim.

To test add to your .vimrc:
```
Plug 'gko/material.vim', { 'branch': 'add_cterm_support' }
```

see #40

#### Normal mode

image

#### Insert mode

image

#### Visual mode

image

#### Replace mode

image

#### Normal mode

image

#### Insert mode

image

#### Visual mode

image

#### Replace mode

image

#### Normal mode

image

#### Insert mode

image

#### Visual mode

image

#### Replace mode

image

#### Normal mode

image

#### Insert mode

image

#### Visual mode

image

#### Replace mode

image

#### Normal mode

image

#### Insert mode

image

#### Visual mode

image

#### Replace mode

image

#### Normal mode

image

#### Insert mode

image

#### Visual mode

image

#### Replace mode

image

#### Normal mode

image

#### Insert mode

image

#### Visual mode

image

#### Replace mode

image

#### Normal mode

image

#### Insert mode

image

#### Visual mode

image

#### Replace mode

image

#### Normal mode

image

#### Insert mode

image

#### Visual mode

image

#### Replace mode

image

#### Normal mode

image

#### Insert mode

image

#### Visual mode

image

#### Replace mode

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,9 @@ let g:airline_theme = 'material'
 
 Corresponding terminal color schemes are included in this repo. You can find them [here](https://github.com/kaicataldo/material.vim/tree/master/terminal-colors/).
 
+#### Vim in macOS
+
+The default version of `vim` on `macOS` doesn't support `termguicolors`. In order for the colorscheme to work properly you need to apply the terminal colors from above. Or you can install the latest version from `brew`.
```
Comment (kaicataldo) [inline README.md:91]: Thanks, this is helpful documentation.

File: colors/material.vim [hunk 2]
```diff
@@ -414,5 +414,26 @@ if has('nvim')
   let g:terminal_color_13 = g:terminal_color_5
   let g:terminal_color_14 = g:terminal_color_6
   let g:terminal_color_15 = g:terminal_color_7
+elseif has('terminal')
```
Comment (kaicataldo) [inline colors/material.vim]: Do you mind explaining why this is necessary?

**Review by kaicataldo — COMMENTED** [review]
One small question for clarification, but otherwise this LGTM. Thanks for working on this!

**Comment by kaicataldo** [comment]
And thanks for the all the screenshots :pray: This is super helpful for posterity!

File: colors/material.vim (hunk 2 above)
Comment (gko) [reply colors/material.vim]: I actually researched this a bit and didn't find it necessary 😄

Although I saw `g:terminal_ansi_colors` defined in many colorschemes.

I can remove it.

Comment (kaicataldo) [reply colors/material.vim]: Sounds good! I can merge this as soon as that's fixed up. Thanks!

Comment (gko) [reply colors/material.vim]: done

**Review by kaicataldo — APPROVED** [review]
Thank you for contributing!

**Pull request merged**